File: async-scala-client/src/main/scala/io/swagger/client/model/ItemCpom.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class ItemCpom (
  companyId: UUID,  // Company ID
cityCode: String,  // IBGE city code
itemCode: String,  // ERP Code
code: String  // Code in City
)

File: javascript-closure-angular-client/API/Client/ItemCpom.js
goog.provide('API.Client.ItemCpom');

/**
 * @record
 */
API.Client.ItemCpom = function() {}

/**
 * Company ID
 * @type {!API.Client.UUID}
 * @export
 */
API.Client.ItemCpom.prototype.companyId;

/**
 * IBGE city code
 * @type {!string}
 * @export
 */
API.Client.ItemCpom.prototype.cityCode;

/**
 * ERP Code
 * @type {!string}
 * @export
 */
API.Client.ItemCpom.prototype.itemCode;

/**
 * Code in City
 * @type {!string}
 * @export
 */
API.Client.ItemCpom.prototype.code;

